### PR TITLE
C4-103 Include Terms Agg

### PR DESCRIPTION
**Pull request by willronchetti**
- Augments our `terms` aggregations to `include` fields requested in the search (so counts are always available)
- Update DP column order `20` -> `40`
- Fix Ingestion Error reporting

### CODE REVIEW

File: src/encoded/ingestion_listener.py [hunk 1]
```diff
@@ -506,18 +506,18 @@ def discard(msg):
                                                  institution=file_meta['institution']['@id'])
                 success, error = variant_builder.ingest_vcf()
 
+                # report results in error_log regardless of status
+                msg = self.ingestion_report.brief_summary()
+                log.error(msg)
```
Comment (netsettler) [inline src/encoded/ingestion_listener.py:511]: Can you explain better why this is not in an info level? Weird to log non-errors as errors. I guess it's OK but it risks cluttering the wrong log and appearing to give the impression there are lots of errors. Could the bug be that too much unwanted .info is given?

File: src/encoded/schemas/variant_sample.json [hunk 2]
```diff
@@ -503,7 +503,7 @@
             "title": "Coverage, VAF",
             "aggregation_type": "stats",
             "number_step": 1,
-            "order": 20,
+            "order": 40,
```
Comment (netsettler) [inline src/encoded/schemas/variant_sample.json:506]: Is the meaning of these orders documented somewhere?

File: src/encoded/search/lucene_builder.py [hunk 3]
```diff
@@ -960,18 +960,22 @@ def _add_range_aggregation(cls, facet, query_field, search_filters, string_query
...
+        """ Builds a terms aggregation, specifically requesting counts for any selected values. """
+        agg = {
             TERMS: {
                 'size': MAX_FACET_COUNTS,
                 'field': query_field,
                 'missing': facet.get('missing_value_replacement', 'No value')
             }
         }
+        if requested_values:  # getall returns [], not None
+            agg[TERMS]['include'] = requested_values
```
Comment (netsettler) [inline src/encoded/search/lucene_builder.py:973]: Should this have a constant?

**Review by netsettler — APPROVED** [review]
No real blockers here, I guess.

File: src/encoded/ingestion_listener.py (hunk 1 above)
Comment (willronchetti) [reply src/encoded/ingestion_listener.py:511]: I want a report of this sent to Sentry on every run for now (so long as we're not running this on the order of hundreds of times a week, which we won't be near for awhile).

File: src/encoded/schemas/variant_sample.json (hunk 2 above)
Comment (willronchetti) [reply src/encoded/schemas/variant_sample.json:506]: It just defines the column ordering WRT one another.

**Pull request merged**